generators/server/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = JhipsterServerGenerator.extend({
                 this.messageBroker = false;
             }
 
+            this.apiFirst = this.config.get('apiFirst') === 'no' ? false : this.config.get('apiFirst');
```

Comment:
The idea is to have it extensible for future possible technologies. Like we have for searchEngine, messageBroker, ...
Swagger has the most hype ATM, but that could change.